Makefile

```diff
@@ -23,6 +24,13 @@ setup-build:
 
 compile: clean cythonize setup-build
 
+install-driver:
+	git submodule update --init --recursive
+	mkdir -p $(current_dir)/vendor/cpp-driver/build
+	cd $(current_dir)/vendor/cpp-driver/build && \
+		cmake -D CASS_BUILD_STATIC=ON -D CMAKE_CXX_FLAGS=-fPIC -D CASS_BUILD_SHARED=OFF -D CASS_USE_STATIC_LIBS=ON -D CMAKE_C_FLAGS=-fPIC .. && \
+		make
```

Comment:
any chance to use this new command here [1]?


[1] https://github.com/pfreixes/acsylla/blob/master/.github/workflows/ci.yml#L36